styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesDockerRunner.java

```diff
@@ -158,7 +160,11 @@ static Pod createPod(WorkflowInstance workflowInstance, RunSpec runSpec) {
         .build());
     env.add(new EnvVarBuilder()
         .withName(TRIGGER_ID)
-        .withValue(runSpec.triggerId().orElse(null))
+        .withValue(runSpec.trigger().map(TriggerUtil::triggerId).orElse(null))
+        .build());
+    env.add(new EnvVarBuilder()
+        .withName(TRIGGER_NAME)
```

Comment:
I know that it's called `TriggerUtil.name`, but I think using the term `name` is misleading. It's actually the trigger  type, e.g. `natural`, `ad-hoc`, `backfill`. `name` sounds like something that the user might have set.

I would use `STYX_TRIGGER_TYPE` on this user facing part and refactor the utilities to also say type instead of name.